cli/src/cli_builder.rs

```diff
@@ -476,6 +476,13 @@ fn xxx<'a>(command: CliCommand) -> Option<&'a str> {
         CliCommand::NetworkingSwitchPortSettingsView => {
             Some("system networking switch-port-settings view")
         }
+        CliCommand::NetworkingSwitchPortLldpConfigView => {
+            Some("system networking lldp config view")
+        }
+        CliCommand::NetworkingSwitchPortLldpConfigUpdate => {
+            Some("system networking lldp config update")
+        }
```

Comment:
```suggestion
        CliCommand::NetworkingSwitchPortLldpConfigView => {
            Some("system networking lldp view")
        }
        CliCommand::NetworkingSwitchPortLldpConfigUpdate => {
            Some("system networking lldp update")
        }
```

Do we think that `config` is an important subcommand or can we just view `lldp` as the thing that we're viewing and updating?

Reply:
My intent was to distinguish between the data that we broadcast to the world (i.e., the `config`) and the data we collect from the world (i.e., the `neighbors`).  If we flatten the `config` command, I'm sure the admin will be able to figure out what's going on, but the added context makes it a little clearer.  That's as firm a stand as I'm prepared to take, and I'm happy to let you decide whether that is important or not.

Reply:
I think "config" would differ from other, similar subcommands. Let's remove it. I expect we'll be reorganizing with @internet-diglett's work...